packages/vue/docs/.vuepress/config.js

```diff
@@ -23,8 +23,69 @@ module.exports = {
         title: 'Components [WIP]',
         collapsable: false,
         children: [
-          ['/components/arrow', 'Arrow'],
-          ['/components/button', 'Button'],
+          // @components-docs-start (keep comment and indentation for auto-generated component docs)
+          ["/components/SfAccordion", "Accordion"],
+          ["/components/SfAccordionItem", "Accordion Item"],
+          ["/components/SfAddToCart", "Add To Cart"],
+          ["/components/SfAlert", "Alert"],
+          ["/components/SfArrow", "Arrow"],
+          ["/components/SfBadge", "Badge"],
+          ["/components/SfBanner", "Banner"],
+          ["/components/SfBannerGrid", "Banner Grid"],
+          ["/components/SfBottomNavigation", "Bottom Navigation"],
+          ["/components/SfBottomNavigationItem", "Bottom Navigation Item"],
```

Comment:
Imho we shouldn't have components from `_internal` folders here as they're just a part of their parents.

Also I'm not sure if we shouldn't keep URLs without Sf- prefix. WDYT?

Reply:
1. Sure I can leave the _internal components out. I guess this would simplify the logic even a bit.
I just thought that users might be interested in their props, slots, etc. (e.g. many `*Item`s have slots; `HeroItem` has props (even with [descriptions](https://deploy-preview-354--docs-storefrontui.netlify.com/components/SfHeroItem.html#props))).
So no _internal components?

2. Sure, I can remove the Sf prefix.

Reply:
This is now solved; see new commits.